src/logcollector/read_win_event_channel.c

```diff
@@ -310,7 +310,7 @@ int update_bookmark(EVT_HANDLE evt, os_channel *channel)
         goto cleanup;
     }
 
-    if ((buffer = calloc(size, sizeof(char))) == NULL) {
+    if (buffer = calloc(size, sizeof(wchar_t)), buffer == NULL) {
```

Comment:
According to the documentation of [EvtRender](https://docs.microsoft.com/en-us/windows/win32/api/winevt/nf-winevt-evtrender), the size of the buffer is measured in bytes.